scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/context/TaskContextVariableExtractor.java

```
@@ -122,11 +122,17 @@
 
     public Map<String, Serializable> extractScopeVariables(TaskContext taskContext) {
         Map<String, Serializable> variables = new HashMap<>();
-
+        Map<String, Serializable> previousVariables = null;
+        try {
+            previousVariables = extractVariables(taskContext, false);
+        } catch (IOException | ClassNotFoundException e) {
+            logger.error("Unable to extract variables", e);
```

Should we throw an exception here instead of just logging an error?